code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -1,177 +0,0 @@
-				if(!M)
-					param = null
-
-				if(param)
-					message = "<B>[src]</B> salutes to [param]."
-				else
-					message = "<B>[src]</b> salutes."
-			m_type = 1
-		if("bow","bows")
-			if(!src.buckled)
-				var/M = null
-				if(param)
-					for(var/mob/A in view(null, null))
-						if(param == A.name)
-							M = A
-							break
-				if(!M)
-					param = null
-
-				if(param)
-					message = "<B>[src]</B> bows to [param]."
-				else
-					message = "<B>[src]</B> bows."
-			m_type = 1
-
-		if("clap","claps")
-			if(!src.restrained())
-				message = "<B>[src]</B> claps."
-				m_type = 2
-		if("flap","flaps")
-			if(!src.restrained())
-				message = "<B>[src]</B> flaps its wings."
-				m_type = 2
-
-		if("aflap","aflaps")
-			if(!src.restrained())
-				message = "<B>[src]</B> flaps its wings ANGRILY!"
-				m_type = 2
-
-		if("twitch")
-			message = "<B>[src]</B> twitches violently."
-			m_type = 1
-
-		if("twitch_s","twitches")
-			message = "<B>[src]</B> twitches."
-			m_type = 1
-
-		if("nod","nods")
-			message = "<B>[src]</B> nods."
-			m_type = 1
-
-		if("deathgasp")
-			message = "<B>[src]</B> shudders violently for a moment, then becomes motionless, its eyes slowly darkening."
-			m_type = 1
-
-		if("glare","glares")
-			var/M = null
-			if(param)
-				for(var/mob/A in view(null, null))
-					if(param == A.name)
-						M = A
-						break
-			if(!M)
-				param = null
-
-			if(param)
-				message = "<B>[src]</B> glares at [param]."
-			else
-				message = "<B>[src]</B> glares."
-
-		if("stare","stares")
-			var/M = null
-			if(param)
-				for(var/mob/A in view(null, null))
-					if(param == A.name)
-						M = A
-						break
-			if(!M)
-				param = null
-
-			if(param)
-				message = "<B>[src]</B> stares at [param]."
-			else
-				message = "<B>[src]</B> stares."
-
-		if("look","looks")
-			var/M = null
-			if(param)
-				for(var/mob/A in view(null, null))
-					if(param == A.name)
-						M = A
-						break
-
-			if(!M)
-				param = null
-
-			if(param)
-				message = "<B>[src]</B> looks at [param]."
-			else
-				message = "<B>[src]</B> looks."
-			m_type = 1
-
-
-		if("law")
-			if(istype(module,/obj/item/robot_module/security))
-				message = "<B>[src]</B> shows its legal authorization barcode."
-
-				playsound(src.loc, 'sound/voice/biamthelaw.ogg', 50, 0)
-				m_type = 2
-			else
-				to_chat(src, "You are not THE LAW, pal.")
-
-		if("halt")
-			if(istype(module,/obj/item/robot_module/security))
-				message = "<B>[src]</B>'s speakers skreech, \"Halt! Security!\"."
-
-				playsound(src.loc, 'sound/voice/halt.ogg', 50, 0)
-				m_type = 2
-			else
-				to_chat(src, "You are not security.")
-
-		if("flip","flips")
-			m_type = 1
-			message = "<B>[src]</B> does a flip!"
-			src.SpinAnimation(5,1)
-
-		if("help")
-			to_chat(src, "salute, bow-(none)/mob, clap, flap, aflap, twitch, twitches, nod, deathgasp, glare-(none)/mob, stare-(none)/mob, look,\n law, halt")
-
-	..(act, m_type, message)
-
-/mob/living/silicon/robot/verb/powerwarn()
```

Comment:
The only thing about this that makes it an emote is the fact that it uses emote CDs, make sure you don't forget to reinclude this